ext/phar/stream.c

```diff
@@ -306,13 +305,11 @@ static php_stream * phar_wrapper_open_url(php_stream_wrapper *wrapper, const cha
 	}
 	php_url_free(resource);
 #ifdef MBO_0
-		fprintf(stderr, "Pharname:   %s\n", idata->phar->filename);
+		fprintf(stderr, "Pharname:   %s\n", idata->phar->fname);
```

Comment:
Intentional change from filename to fname?

Reply:
Yes, this debug code is so old that when the field name was changed the debug code was forgotten